package.json

```diff
@@ -29,11 +29,14 @@
     "translations:collect": "i18next-scanner 'src/**/*.{js,jsx}'",
     "translations:check": "node scripts/checkTranslations.js",
     "relay": "relay-compiler --src ./src --schema ./schema.graphql",
-    "schema": "graphql get-schema dev"
+    "schema": "graphql get-schema dev",
+    "commitmsg": "commitlint -E GIT_PARAMS"
```

Comment:
This is a pre-commit hook?

Reply:
Yes and also merge https://git-scm.com/docs/githooks#_commit_msg

Reply:
Recommended by commitlint documentation here http://marionebl.github.io/commitlint/#/guides-local-setup?id=install-husky